app/models/company.rb

```diff
@@ -6,6 +6,9 @@ class Company < ActiveRecord::Base
   validates :title,           presence: true
   validates :description,     presence: true
   validates :about,           presence: true
-  validates :address,         presence: true
+  validates :address1,        presence: true
+  validates :city,            presence: true
+  validates :state,           presence: true
+  validates :zip,             presence: true
```

Comment:
@godfreym no one uses the extensions except for the post office. 5 digits is standard